fix: layout header to properly handle long text without breaking the layout

mooka95 opened the pull request:
#24190

<!--
Hello 👋 Thank you for submitting a pull request.

To help us merge your PR, make sure to follow the instructions below:

- Create or update the tests
- Create or update the documentation at https://github.com/strapi/documentation
- Refer to the issue you are closing in the PR description: Fix #issue
- Specify if the PR is ready to be merged or work in progress (by opening a draft PR)

Please ensure you read the Contributing Guide: https://github.com/strapi/strapi/blob/main/CONTRIBUTING.md
-->

### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

mooka95 commented:
@derrickmehaffy

mooka95 commented:
@Sija

derrickmehaffy commented:
We don't place a max beyond what the database supports.

derrickmehaffy commented:
Mmmmm it's ugly but I don't see how it would hurt anything. Realistically this should be solved in the design system to just trim the view.

Imposing limits to do what the presentation layer should be doing just ends up with the weird restrictions no one understands why they exist (we have plenty of those already 😂)

mooka95 commented:
@derrickmehaffy @Sija reverted validation of first and last name  i added and fixed layout header to fit any long text 
<img width="1783" height="912" alt="image" src="https://github.com/user-attachments/assets/f4adae17-2ea2-44b1-8105-b988de73b951" />
<img width="1900" height="935" alt="image" src="https://github.com/user-attachments/assets/b3996c45-6028-4fe2-a468-e6c0a02c9e09" />
and this fixed profile page and Dashboard

mooka95 commented:
@hanpaine  fixed conflict

Adzouz reviewed (approved):
Looks good to me, thanks for the contribution! 🙏
